src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
Yes. I (as usual) have a branch with all this but afraid to pull it in since it will break all existing addins.